packages/legend-art/style/base/_panel.scss

```diff
@@ -251,6 +290,15 @@
     }
   }
 
+  &__section__selector {
```

Comment:
looks like we don't need these classes anymore, let's remove them.

Reply:
Selector and remove-btn is needed for other PP styling, but there is probably a better method I will add componentize later if that is ok?
<img width="596" alt="image" src="https://user-images.githubusercontent.com/41248956/189765254-69fc1048-9a25-41a7-9bf9-fc7f363176eb.png">

Reply:
Ohhh this is in the legendary contents but I guess like all the other stuff it makes sense then that it should be exclusively in there! I will remove 👍